hw/ip/hmac/dv/hmac_sim_cfg.hjson

```diff
@@ -113,9 +113,21 @@
     }
 
     {
-      name: hmac_test_hmac_vectors
+      name: hmac_test_hmac256_vectors
       uvm_test_seq: hmac_test_vectors_hmac_vseq
-      run_opts: ["+test_vectors_dir={build_dir}/src/lowrisc_dv_test_vectors_0 +sha2_digest_size=SHA2_256"]
+      run_opts: ["+test_vectors_dir={build_dir}/src/lowrisc_dv_test_vectors_0 +test_timeout_ns=1_000_000_000 +sha2_digest_size=SHA2_256"]
+    }
+
+    {
+      name: hmac_test_hmac384_vectors
+      uvm_test_seq: hmac_test_vectors_hmac_vseq
+      run_opts: ["+test_vectors_dir={build_dir}/src/lowrisc_dv_test_vectors_0 +test_timeout_ns=2_000_000_000 +sha2_digest_size=SHA2_384"]
```

Comment:
@martin-velay I extend the timeout here for 384 and 512, I guess this should cover it